lib/api/handler/pokemon_api.dart

```diff
@@ -9,5 +13,19 @@ class PokemonApi {
   final ApiClient apiClient;
   final Uri baseUri;
 
-//TODO: to be continued...
-}
+  Future<List<Pokemon>> getPokemonList({
+    required String offset,
+    required String limit,
+  }) async {
+    final queryParameters = <String, dynamic>{};
+    queryParameters[queryParamLimit] = limit;

```

Comment:
I don't see anything related to your `queryParamLimit` under your string_constant.dart file, is this even existing? 🤔